slack-bot/tests/test_message_handlers.py

```
@@ -0,0 +1,312 @@
+        text = "Hello"
+        user_id = "U12345"
+        channel = "C12345"
+        thread_ts = "1234567890.123456"
+        
+        mock_slack_service.send_thinking_indicator = AsyncMock(return_value="thinking_ts")
+        mock_slack_service.get_thread_history = AsyncMock(side_effect=Exception("API error"))
+        mock_slack_service.delete_thinking_indicator = AsyncMock()
+        mock_slack_service.client = MagicMock()  # Add client attribute
+        
+        with patch('handlers.message_handlers.handle_error') as mock_handle_error:
+            mock_handle_error = AsyncMock()
```

Ruff (0.11.9)

Local variable `mock_handle_error` is assigned to but never used

Remove assignment to unused variable `mock_handle_error`

(F841)